crates/era-cheatcodes/src/cheatcodes.rs

```diff
@@ -296,13 +339,50 @@ impl CheatcodeTracer {
                 tracing::info!("👷 Returndata is {:?}", account_nonce);
                 self.return_data = Some(vec![account_nonce]);
             }
+            getRecordedLogs(getRecordedLogsCall {}) => {
+                tracing::info!("👷 Getting recorded logs");
+                let log_vector: Vec<Log> = self
```

Comment:
We can simply call this `logs`. We should avoiding naming things after their types as it is easily inferred in strongly typed languages.